### PR TITLE
Implementing closing date for Election

### PR DESCRIPTION
**Pull request by davidwessman**
The Election needs another date, one to stop new candidates for posts elected at the General meeting, another one for closing the election all together.

Solves #171

### CODE REVIEW

**Comment by davidwessman** [comment]
Okay if I merge this?

File: db/schema.rb [hunk 1]
```diff
@@ -429,14 +364,11 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.text     "comment",     limit: 65535
-    t.string   "status",      limit: 255,   default: "Ej bestämd"
+    t.string   "status",      limit: 255,   default: "Ej bestÃƒÂ¤md"
```
Comment (henrikssn) [inline db/schema.rb]: Fix encoding

**Comment by henrikssn** [comment]
Tests?

**Comment by henrikssn** [comment]
LGTM

Fix hound comments and then merge this.

**Comment by SirDirac** [comment]
This PR was deployed to Production. Reference: <a href='http://samson.fsektionen.se/projects/web/deploys/100' target='_blank'>f79c6fd</a>

**Pull request merged**